packaging/installer/UNINSTALL.md

```diff
@@ -33,9 +33,21 @@ NETDATA_ADDED_TO_GROUPS="<additional groups>"  # Additional groups for a user ru
 
 3.  Run `netdata-uninstaller.sh` as follows
 
-```sh
-${NETDATA_PREFIX}/usr/libexec/netdata/netdata-uninstaller.sh --yes --env <environment_file>
-```
+    3.1 Interactive mode (Default)
+
+    The default mode in the uninstaller script is `interactive`. This means that the script provides to the user the option to reply with "yes" (`y`/`Y`) or "no" (`n`/`N`) to control the removal of each netdata asset in the filesystem.
```

Comment:
```suggestion

    The default mode in the uninstaller script is **interactive**. This means that the script provides the user the option to reply with "yes" (`y`/`Y`) or "no" (`n`/`N`) to control the removal of each Netdata asset in the filesystem.
```

I think if you want to highlight "interactive" it is better to use bold font. Because why inline code? 🤔